analyses/independent-samples/02-generate-independent-rnaseq.R

```diff
@@ -54,12 +60,19 @@ if (!dir.exists(out_dir)){
   dir.create(out_dir, recursive = TRUE)
 }
 
-rnaseq_primary_file <- file.path(out_dir, 
-                                  "independent-specimens.rnaseq.primary.tsv")
-rnaseq_relapse_file <- file.path(out_dir, 
-                                  "independent-specimens.rnaseq.relapse.tsv")
-rnaseq_primplus_file <- file.path(out_dir, 
-                                  "independent-specimens.rnaseq.primary-plus.tsv")
+rnaseq_primary_each_file <- file.path(out_dir, 
+                                  "independent-specimens.rnaseq.primary.eachcohort.tsv")
+rnaseq_relapse_each_file <- file.path(out_dir, 
+                                  "independent-specimens.rnaseq.relapse.eachcohort.tsv")
+rnaseq_primplus_each_file <- file.path(out_dir, 
+                                  "independent-specimens.rnaseq.primary-plus.eachcohort.tsv")
+
+rnaseq_primary_all_file <- file.path(out_dir, 
+                                 "independent-specimens.rnaseq.primary.allcohorts.tsv")
+rnaseq_relapse_all_file <- file.path(out_dir, 
+                                 "independent-specimens.rnaseq.relapse.allcohorts.tsv")
+rnaseq_primplus_all_file <- file.path(out_dir, 
+                                  "independent-specimens.rnaseq.primary-plus.allcohorts.tsv")
```

Comment:
Similarly, for backward compatibility.

```suggestion
rnaseq_primary_all_file <- file.path(out_dir, 
                                 "independent-specimens.rnaseq.primary.tsv")
rnaseq_relapse_all_file <- file.path(out_dir, 
                                 "independent-specimens.rnaseq.relapse.tsv")
rnaseq_primplus_all_file <- file.path(out_dir, 
                                  "independent-specimens.rnaseq.primary-plus.tsv")
```